tests/unit/dojson/test_dojson_hep.py

```diff
@@ -437,20 +437,20 @@ def test_succeeding_entry(marcxml_to_json, json_to_marc):
 
 def test_url(marcxml_to_json, json_to_marc):
     """Test if url is created correctly."""
-    assert (marcxml_to_json['urls'][0]['url'] ==
+    assert (marcxml_to_json['urls'][0]['value'] ==
             json_to_marc['8564'][0]['u'])
-    assert (marcxml_to_json['urls'][0]['size'] ==
-            json_to_marc['8564'][0]['s'])
-    assert (marcxml_to_json['urls'][0]['doc_string'] ==
-            json_to_marc['8564'][0]['w'])
+    #assert (marcxml_to_json['urls'][0]['size'] ==
+            #json_to_marc['8564'][0]['s'])
+    #assert (marcxml_to_json['urls'][0]['doc_string'] ==
+            #json_to_marc['8564'][0]['w'])
```

Comment:
Yes, I noticed that a later commit deletes these comments.

A better way to do it is modifying the commit itself, rather than adding a commit later to fix a previous commit: doing that creates a dependency between two commits and violates their atomicity, which makes integration harder.